### PR TITLE
Multiple signatures

### PR DESCRIPTION
**Pull request by badurinantun**
## Usage related changes
- Fix signature arguments not accepting multiple signatures

## Development related changes
- None

# Checklist:

- [x] I have formatted the code
- [x] I have performed a self-review of the code
- [x] I have rebased to the base branch
- [x] I have documented the changes
- [x] I have updated the tests
- [x] I have created a PR to the `plugin` branch of [`starknet-hardhat-example`](https://github.com/Shard-Labs/starknet-hardhat-example).

### CODE REVIEW

File: test/general-tests/signature-argument/check.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+echo "Address: $address"
+
+# test call
+call_output=$(npx hardhat starknet-call \
+    --contract "signatures" --address "$address" \
+    --function "get_signature" --starknet-network "$NETWORK" \
+    --signature "$signature")
+
+echo $call_output
```
Comment (FabijanC) [inline test/general-tests/signature-argument/check.sh:26]: Did you leave the log lines such as this one on purpose?

File: test/general-tests/signature-argument/check.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+if [[ "$call_output" != *"$signature_len $signature"* ]]; then
```
Comment (FabijanC) [inline test/general-tests/signature-argument/check.sh:28]: Why do pattern matching with the inequality operator?

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: test/general-tests/signature-argument/check.sh (hunk 1 above)
Comment (badurinantun) [reply test/general-tests/signature-argument/check.sh:26]: Yes

File: test/general-tests/signature-argument/check.sh (hunk 2 above)
Comment (badurinantun) [reply test/general-tests/signature-argument/check.sh:28]: [e042f33](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/109/commits/e042f332aa977ff629e807f644bfe02a461996c9)

**Review by FabijanC — APPROVED** [review]

**Pull request merged**